### PR TITLE
fix: resolve duplicate rsyslog entries for Azure Linux

### PR DESCRIPTION
**Pull request by henryli001**
**What type of PR is this?**

**What this PR does / why we need it**:

This PR aims to fix the issue of duplicate entries in /var/log/messages file in Azure Linux AKS. 

- The default rsyslog configuration file (/etc/rsyslog.conf) in Azure Linux contains the following setting:
`*.*,auth,authpriv,mail,cron.none    -/var/log/messages`

- The 60-CIS.conf file provided by AgentBaker contains the following setting:
`*.*;mail.none;news.none            -/var/log/messages`

The duplicate specification of `*.*` is the reason why logs are written twice into /var/log/messages. The default setting in Azure Linux cannot be modified as that will affect the standard Azure Linux images and thus this PR makes the fix in the AgentBaker repo to not apply `*.*` twice for Azure Linux AKS container host.

Bug link: https://microsoft.visualstudio.com/OS/_workitems/edit/50154304/

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: vhdbuilder/packer/pre-install-dependencies.sh [hunk 1]
```diff
@@ -48,6 +48,18 @@ stop_watch $capture_time "Set Permissions if Mariner" false
 start_watch
 
 copyPackerFiles
+
+# Update rsyslog configuration
+RSYSLOG_CONFIG_FILEPATH="/etc/rsyslog.d/60-CIS.conf"
+if [[ $OS == $MARINER_OS_NAME ]]; then
+    echo -e "\nnews.none                          -/var/log/messages" >> ${RSYSLOG_CONFIG_FILEPATH}
```
Comment (cameronmeissner) [inline vhdbuilder/packer/pre-install-dependencies.sh:55]: do we actually need all this blank space?

Comment (henryli001) [reply vhdbuilder/packer/pre-install-dependencies.sh:55]: This blank space is added to ensure that this line aligns with the spacing of other lines in 60-CIS.conf file.

**Review by dallasd1 — APPROVED** [review]

**Comment by cameronmeissner** [comment]
lgtm pending green abe2e run

**Comment by henryli001** [comment]
@cameronmeissner all checks including AgentBaker E2E has passed.

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**